cli/azd/pkg/input/console.go

```diff
@@ -343,6 +374,7 @@ func NewConsole(noPrompt bool, isTerminal bool, w io.Writer, handles ConsoleHand
 		defaultWriter: w,
 		writer:        w,
 		formatter:     formatter,
+		consoleWidth:  getConsoleWidth(),
```

Comment:
Isn't this fixed? When the user resizes the console, shouldn't we be updating the enforced length?

Reply:
Ideas for handling real-time updates:
1. Poll console size, update spinner text if needed (check if spinner is active, update text)
2. Figure out os-specific ways of getting a console size update callback, do the same thing about updating spinner text

Since we can't directly update the animation logic yet.

Reply:
> Poll console size

The spinner library has a frequency of 200 ms to refresh the spinner.  We would need to poll in a higher frequency to be responsive.  I don't think the polling at such frequency is a good idea, because we are hitting an OS api, and I don't know if it could ever take longer than the frequency.

The ideal thing would be what you mentioned on 2).  Have a push event when the size changes.  

But I don't think this is worth it..  I don't think terminal resize is a thing that happens a lot.  
I think the common case is when the terminal/window is moved and gets auto sized while it is drag/drop.

Reply:
We can have a lower polling frequency. It's okay to be slow to respond, it's not okay to not respond.

My take on performance is always to profile (we never know what 3rd party code actually does).

Otherwise we get this:

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/2322434/234390858-e6af8f11-9de9-4b16-a7b7-734f92498e07.png">

Reply:
Can we wait until we know that we really need the real-time responsiveness?  
I would probably like to create an alpha feature `responsive-console` that uses the extra polling routine to handle this and let folks to opt-in.